src/store/session-manager.js

```diff
@@ -65,6 +65,10 @@ var SessionManager = function (managerOptions) {
             Object.keys(keyNames).forEach(function (cookieKey) {
                 var cookieName = keyNames[cookieKey];
                 store.remove(cookieName);
+                document.cookie.split('; ')
+                    .map((c)=> c.split('=')[0] || '')
+                    .filter((c)=> c.startsWith(cookieName))
```

Comment:
Any reason to use `startsWith` over `indexOf` here?  `indexOf` is faster in most browsers and has wider support (IE doesn't support `startsWith`).

Reply:
No reason, I did see that IE wasn't supported and I was fine with that 🙂 ; didn't know `indexOf` was faster though! Let me revise.